pilco/models/smgpr.py

```diff
@@ -4,51 +4,53 @@
 
 from .mgpr import MGPR
 
-float_type = gpflow.settings.dtypes.float_type
+from gpflow import config
+float_type = config.default_float()
 
 
 class SMGPR(MGPR):
-    def __init__(self, X, Y, num_induced_points, name=None):
-        gpflow.Parameterized.__init__(self, name)
+    def __init__(self, data, num_induced_points, name=None):
         self.num_induced_points = num_induced_points
-        MGPR.__init__(self, X, Y, name)
+        MGPR.__init__(self, data, name)
 
-    def create_models(self, X, Y):
+    def create_models(self, data):
         self.models = []
         for i in range(self.num_outputs):
-            kern = gpflow.kernels.RBF(input_dim=X.shape[1], ARD=True)
+            kern = gpflow.kernels.SquaredExponential(lengthscales=tf.ones([data[0].shape[1],], dtype=float_type))
             Z = np.random.rand(self.num_induced_points, self.num_dims)
             #TODO: Maybe fix noise for better conditioning
-            self.models.append(gpflow.models.SGPR(X, Y[:, i:i+1], kern, Z=Z))
-            self.models[i].clear(); self.models[i].compile()
-    
+            self.models.append(gpflow.models.GPRFITC((data[0], data[1][:, i:i+1]), kern, inducing_variable=Z))
```

Comment:
Is this a different type of sparse GP? If so, we should rename the class.

Reply:
My understanding is that this was the default in the previous gpflow version, just needs to be specified explicitly in the newer one.